src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/Endpoint/CatalogEndpoint.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class CatalogEndpoint : IEndpoint
+    {
+        public bool Enabled => true;
+
+        public ReadCursor Cursor => new MemoryCursor(MemoryCursor.MaxValue);
```

Comment:
Let's add a quick comment why we use `MaxValue` here

Reply:
done